pkg/testutil/testutil.go

```diff
@@ -70,6 +71,7 @@ func WithStdin(r io.Reader) func(*Cmd) {
 func (b *Base) Cmd(args ...string) *Cmd {
 	icmdCmd := icmd.Command(b.Binary, append(b.Args, args...)...)
 	icmdCmd.Env = b.Env
+	icmdCmd.Dir = b.Dir
```

Comment:
what's the behavior if we don't set `Dir` (which means both b.Dir and icmdCmd.Dir are "")?

Reply:
> what's the behavior if we don't set `Dir` (which means both b.Dir and icmdCmd.Dir are "")?

I believe we then execute in whatever is `os.Getwd`.

Reply:
https://go.dev/play/p/x_-0-uNtVd-